### PR TITLE
Add dotnet-remove-p2p

### PR DESCRIPTION
**Pull request by krwq**
Fix subitem of https://github.com/dotnet/cli/issues/4746

### CODE REVIEW

File: src/dotnet/MsbuildProjectExtensions.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.DotNet.Tools.Add.ProjectToProjectReference
```
Comment (krwq) [inline src/dotnet/MsbuildProjectExtensions.cs]: This file is missing header

File: src/dotnet/dotnet.csproj [hunk 2]
```diff
@@ -1,6 +1,5 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
-
```
Comment (krwq) [inline src/dotnet/dotnet.csproj]: revert this change - VS did it again

File: src/dotnet/MsbuildProjectExtensions.cs (hunk 1 above)
Comment (krwq) [inline src/dotnet/MsbuildProjectExtensions.cs]: done

File: src/dotnet/dotnet.csproj (hunk 2 above)
Comment (krwq) [inline src/dotnet/dotnet.csproj]: pending

**Comment by krwq** [comment]
@dotnet-bot test this please

**Comment by krwq** [comment]
@MattGertz, can I get approval on this? This is adding subitem listed in of https://github.com/dotnet/cli/issues/4746

**Comment by MattGertz** [comment]
@krwq can you specify what this fixes?  I see the commits above but I'm not clear on the customer scenarios.  Thanks!

**Comment by krwq** [comment]
@MattGertz this is adding feature so that customer can remove p2p references to an msbuild project without opening the file. There are very minimal changes to the existing features (help text, dispatching of new command etc)

**Comment by MattGertz** [comment]
Thanks, approved.
